src/librustc_metadata/decoder.rs

```diff
@@ -572,33 +599,41 @@ impl<'a, 'tcx> CrateMetadata {
         // to support recursive structures
         for variant in &adt.variants {
             for field in &variant.fields {
-                debug!("evaluating the type of {:?}::{:?}", variant.name, field.name);
+                debug!("evaluating the type of {:?}::{:?}",
+                       variant.name,
+                       field.name);
                 let ty = self.get_type(field.did.index, tcx);
                 field.fulfill_ty(ty);
                 debug!("evaluating the type of {:?}::{:?}: {:?}",
-                       variant.name, field.name, ty);
+                       variant.name,
+                       field.name,
+                       ty);
```

Comment:
(same here -- I think this is more readable as it was with these arguments on one line)